contracts/Comptroller/Diamond/facets/PolicyFacet.sol

```diff
@@ -93,6 +116,7 @@ contract PolicyFacet is IPolicyFacet, XVSRewardsHelper {
      * @param redeemTokens The number of tokens being redeemed
      */
     function redeemVerify(address vToken, address redeemer, uint256 redeemAmount, uint256 redeemTokens) external {
+        ensureUserAllowed(redeemer);
```

Comment:
I think this is not needed, because we are already checking the permission for minting in the `redeemAllowed` check (the "pre-hook")